src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -186,7 +186,7 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     for {
       _ <- assertWorkflowExistence(workflowId)
       callToStatusMap <- dataAccess.getExecutionStatuses(workflowId)
-      callToLogsMap <- Future.fromTry(logMapFromStatusMap(callToStatusMap))
+      callToLogsMap <- Future.fromTry(logMapFromStatusMap(callToStatusMap mapValues(_.executionStatus)))
```

Comment:
I could be wrong, but I believe current conventions are either `.mapValues(…)`, or no dot and ` mapValues { … }`?